CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.7.4] (unreleased)
+
+- Added support for the `--application` (or `-a`) option to support AtomVM OTP applications.
+
+### Changed
+
+- Using the `-s init` option is still supported but deprecated.  Use the `--application` (or `-a`) option to generate OTP applications using AtomVM.
```

Comment:
I'm not sure we should depreciate the `-s init` option. I'm thinking of cases where it might be helpful to have a sever and client included in the same project (like the lora_sender and lora_receiver examples from atomvm_lib), or possibly some tcp or udp applications that have a server/client pair. We may want to keep this option available as a convenience.

Reply:
Sorry, `-s` is still supported -- it's the combination of `-s` and `init` that is deprecated (which required a dependency on `atomvm_lib`, where `init` was defined) in favor of using `-a`, which generates a shim that call `init` (which is now being moved into AtomVM).

Reply:
Ok, that's great. I should have looked closer that `-s start_module` doesn't emit a depreciation warning ;)